include/cscore.h

```diff
@@ -1,12 +1,12 @@
 /*----------------------------------------------------------------------------*/
-/* Copyright (c) FIRST 2016. All Rights Reserved.                             */
+/* Copyright (c) 2016-2017 FIRST. All Rights Reserved.                        */
 /* Open Source Software - may be modified and shared by FRC teams. The code   */
 /* must be accompanied by the FIRST BSD license file in the root directory of */
 /* the project.                                                               */
 /*----------------------------------------------------------------------------*/
 
-#ifndef CSCORE_H_
-#define CSCORE_H_
+#ifndef INCLUDE_CSCORE_H_
+#define INCLUDE_CSCORE_H_
```

Comment:
I think Google's style guide thought on this was that the header files get put into a subdirectory named the project name.  E.g. wpiutil would have `src/main/native/include/wpiutil/support/basename.h` instead, note the extra "wpiutil").  We don't do this however, and it would be horribly breaking to change now.